wear/src/main/java/chat/rocket/android/util/UserHelper.kt

```diff
@@ -0,0 +1,47 @@
+package chat.rocket.android.util
+
+import chat.rocket.android.server.GetCurrentServerInteractor
+import chat.rocket.android.server.LocalRepository
+import chat.rocket.common.model.User
+import javax.inject.Inject
+
+class UserHelper @Inject constructor(
```

Comment:
Duplicated. Please remove. You can put this class in the `util` module and use it from there on the `wear` and `app` module as well.